crm_timesheet/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,8 @@
+* `Tecnativa <https://www.tecnativa.com>`_
+  * Antonio Espinosa
```

Comment:
Sorry, this is not your fault, but I'm seeing this requires an extra line in between this line and the above for proper RST syntax.